Source/Scene/Camera.js

```diff
@@ -361,6 +362,17 @@ Camera.prototype._updateCameraChanged = function () {
 
   var percentageChanged = camera.percentageChanged;
 
+  if (!defined(camera._changedHeading)) {
+    camera._changedHeading = camera.heading;
+  }
+
+  var headingChanged = camera._changedHeading === camera.heading ? false : true;
```

Comment:
We talked a bit offline about this, but should there be a `percentChangedHeading`?

Reply:
I think for this pull request, I would like to start by simply implementing the basic functionality. I would love to open up another pull request that gives users access to setting percentage thresholds for:
- heading
- height
- direction

What are your thoughts?

Reply:
Actually stepping back, it seems like the percentage changed is the change since the last time the event was raised, not the change since the last frame. Heading should probably have the same behavior.

I could see some benefit of splitting it out into separate percentage thresholds but I agree that we should get the basic functionality first and then consider whether it's worth doing.

Reply:
This should be taken care of in my most recent commits :+1: